src/node.cc

```diff
@@ -3912,8 +3900,6 @@ static void ParseArgs(int* argc,
       force_repl = true;
     } else if (strcmp(arg, "--no-deprecation") == 0) {
       no_deprecation = true;
-    } else if (strcmp(arg, "--napi-modules") == 0) {
-      load_napi_modules = true;
```

Comment:
I think that would be a good idea.  It  will give us time to update docs etc and will avoid breaking people who are already using it.